tests/sentry/migrations/test_0647_apitoken_add_hashed_columns.py

```diff
@@ -0,0 +1,31 @@
+from django.db import router
+
+from sentry.silo import unguarded_write
+from sentry.testutils.cases import TestMigrations
+from sentry.testutils.silo import no_silo_test
+
+
+@no_silo_test
+class AddHashedColumnsApiToken(TestMigrations):
```

Comment:
No real need for a test when we're just adding a column, mostly we use this to validate backfill logic. The orm validates that this migration runs as part of ci

Reply:
I thought so, but for some reason CI is failing on this? I added the test to confirm my understanding, but I'm still lost.

See:
https://github.com/getsentry/sentry/actions/runs/7924490220/job/21636142797?pr=65300

Reply:
It's because this migration imports a model https://github.com/getsentry/sentry/blob/e1cf672a37a0d2bd8f8dc69cacc9c6faed44c5a2/src/sentry/migrations/0632_apitoken_backfill_last_chars.py#L10

You need to modify it to use `ApiToken = apps.get_model("sentry", "ApiToken")`